mlflow/R/mlflow/tests/testthat/test-tracking-runs.R

```diff
@@ -205,7 +205,7 @@ test_that("mlflow_log_metric() rounds step and timestamp inputs", {
   }
   expect_setequal(
     mlflow_get_metric_history("timestamp_metric")$timestamp,
-    purrr::map(round(timestamp_inputs), mlflow:::milliseconds_to_date)
+    purrr::map_vec(round(timestamp_inputs), mlflow:::milliseconds_to_date)
```

Comment:
`purrr::map` converts each element in the first argument to a list, but `purrr::map_vec` doesn't.